implants/eldritch/src/assets.rs

```diff
@@ -0,0 +1,70 @@
+mod copy_impl;
+mod list_impl;
+
+use derive_more::Display;
+
+use starlark::environment::{Methods, MethodsBuilder, MethodsStatic};
+use starlark::values::none::NoneType;
+use starlark::values::{StarlarkValue, Value, UnpackValue, ValueLike, ProvidesStaticType};
+use starlark::{starlark_type, starlark_simple_value, starlark_module};
+
+use serde::{Serialize,Serializer};
+use rust_embed::RustEmbed;
+
+#[cfg(debug_assertions)]
+#[derive(RustEmbed)]
+#[folder = "../../tests/embedded_files_test"]
+pub struct Asset;
+
+#[cfg(not(debug_assertions))]
+#[derive(RustEmbed)]
+#[folder = "../../implants/golem/embed_files_golem_prod"]
+pub struct Asset;
```

Comment:
I would like to find a better way to define the assets structure.
Currently it gets defined for a test run & and a build.
It also gets defined here in eldritch and in golem.